src/locales/ar-AE.json

```diff
@@ -185,10 +195,12 @@
     "popularBases": "شائع:",
     "portfolio": "ملف",
     "preview": "معاينة",
+    "previewETHUnwrap": "معاينة ETH Unwrap",
```

Comment:
Thanks for the translation :)

I'm actually removing the hardcoded "ETH" in the translation (in favour of passing it as a variable (see here - I actually got an upcoming PR removing a lot of the hardcoded "ETH" stuff (https://github.com/balancer-labs/frontend-v2/pull/699)

Reply:
Ok; do you want me to wait until 699's merged, then adjust this one?